client/src/store.tsx

```diff
@@ -10,6 +10,7 @@ export const store = configureStore({
     player: playerReducer,
     answers: answersReducer,
     question: questionReducer,
+    health: healthReducer
```

Comment:
I think it may be better to hold the player's current health and score in the player structure since they're related, but I can also change it later. If we are going to add a lot of elements to health then it would be worth keeping it separate.